hub/apps/get-started/uno-simple-photo-viewer.md

```diff
@@ -0,0 +1,931 @@
+                    <Grid.RowDefinitions>
+                        <RowDefinition />
+                        <RowDefinition Height="Auto" />
+                    </Grid.RowDefinitions>
+
+                    <Image x:Name="ItemImage"
+                           Source="Assets/StoreLogo.png"
+                           Stretch="Uniform" />
+
+                    <StackPanel Orientation="Vertical"
+                                Grid.Row="1">
+                        <TextBlock Text="{x:Bind ImageTitle}"
+                                   HorizontalAlignment="Center"
+                                   Style="{StaticResource SubtitleTextBlockStyle}" />
+                        <StackPanel Orientation="Horizontal"
+                                    HorizontalAlignment="Center">
+                            <TextBlock Text="{x:Bind ImageFileType}"
+                                       HorizontalAlignment="Center"
+                                       Style="{StaticResource CaptionTextBlockStyle}" />
+                            <TextBlock Text="{x:Bind ImageDimensions}"
+                                       HorizontalAlignment="Center"
+                                       Style="{StaticResource CaptionTextBlockStyle}"
+                                       Margin="8,0,0,0" />
+                        </StackPanel>
+
+                        <RatingControl Value="{x:Bind ImageRating}" 
+                                       IsReadOnly="True"/>
+                    </StackPanel>
+                </Grid>
+            </DataTemplate>
+
+            <Style x:Key="ImageGridView_ItemContainerStyle"
+                   TargetType="GridViewItem">
+                <Setter Property="Background" 
+                        Value="Gray"/>
+                <Setter Property="Margin" 
+                        Value="8"/>
+            </Style>
+
+            <ItemsPanelTemplate x:Key="ImageGridView_ItemsPanelTemplate">
+                <ItemsWrapGrid Orientation="Horizontal"
+                               HorizontalAlignment="Center"/>
+            </ItemsPanelTemplate>
+        </Grid.Resources>
+
+        <GridView x:Name="ImageGridView"
+                  ItemsSource="{x:Bind Images}"
+                  ItemTemplate="{StaticResource ImageGridView_ItemTemplate}"
+                  ItemContainerStyle="{StaticResource ImageGridView_ItemContainerStyle}"
+                  ItemsPanel="{StaticResource ImageGridView_ItemsPanelTemplate}"
+                  ContainerContentChanging="ImageGridView_ContainerContentChanging">
+        </GridView>
+    </Grid>
+</Page>
+```
+
+Recall that the desktop solution also had a `MainWindow.xaml.cs` file that contained code-behind which corresponds to the view. In the Uno Platform project, the code-behind for the `MainPage` view we've copied into is contained in the `MainPage.xaml.cs` file. 
+
+To bring this codebehind multiplatform, we should first move the following into the `MainPage.xaml.cs` file:
```

Comment:
```suggestion
To bring this code-behind multi-platform, we should first move the following into the `MainPage.xaml.cs` file:
```

"code-behind" * "multi-platform" are hyphenated throughout the rest of the document